setup.py

```diff
@@ -9,7 +9,7 @@
 from setuptools import setup
 
 PKG_NAME = "python-doctr"
-VERSION = os.getenv("BUILD_VERSION", "0.6.0a0")
+VERSION = os.getenv("BUILD_VERSION", "0.7.0a0")
```

Comment:
0.6.1a0 out of safety ;)

Reply:
yeah i set it to 0.7.0 because we have no backlog. But you are right :)